templates/CRM/Price/Page/table.tpl

```diff
@@ -79,3 +84,9 @@
 </table>
 {/if}
 {/foreach}
+{if $showGenericMessage}
+  {if $action neq 8}
+    {* We don't have to do anything for delete action because the calling tpl already displays something. *}
+    {ts}This price set is used by at least one contribution, but is not used by any active events or contribution pages or event templates.{/ts}
```

Comment:
Technically could be a contribution or participant

Reply:
Thanks @larssandergreen. I tried to re-use the wording that was used in all the other messages. And in the future it might be even more things, which is also one reason I was trying to get rid of the hardcode of all the possible array keys in multiple places (e.g. the addition of event template missed some places, so it would likely happen again).

I'm open to changing it but then maybe to something more generic. What it's actually checking is usage in line items and in civicrm_price_set_entity.

Reply:
Maybe just "is used at least one record" or "is used by at least one existing record"? Or contribution, not a big deal.